packages/electron-snowpack/lib/get-esbuild-config.js

```diff
@@ -30,8 +30,8 @@ module.exports = async (more) => {
   return {
     platform: 'node',
     format: 'cjs',
-    entryPoints: await promisify(glob)('src/main/index.[jt]s'),
-    outfile: path.join(config.outputDir, 'main/index.js'),
+    entryPoints: await promisify(glob)('src/main/**/*.ts'),
```

Comment:
This would glob _all_ files, potentially including more files than needed. Would it be sufficient to glob for the well-known names like `index` and `preload` only? Also, we can keep the `.js` support.
```suggestion
    entryPoints: await promisify(glob)('src/main/@(index|preload).[jt]s'),
```
See: https://www.npmjs.com/package/glob for the `@(|)` glob pattern.

Reply:
I agree with your opinion. I think the further updates of Electron make the role of the main process due to the discouragement in use of `nodeIntegration` but it would be better to support only two files just for now.